common/variant/src/main/java/org/apache/spark/types/variant/VariantUtil.java

```diff
@@ -377,11 +402,51 @@ public static long getLong(byte[] value, int pos) {
       case TIMESTAMP:
       case TIMESTAMP_NTZ:
         return readLong(value, pos + 1, 8);
+      case YEAR_MONTH_INTERVAL:
```

Comment:
Oh, I see how you updated `getLong`. Could you update the function comments to include details about the new behavior?